doc/library_mlds/base_bigstring.mld

```diff
@@ -0,0 +1,2 @@
+{0 Base_bigstring}
```

Comment:
These pages are too hard to create and delete. The driver should generate them instead.